src/abyssal_tome/app.py

```
@@ -1,729 +1,88 @@
+    if ruling is None:
+        raise HTTPException(status_code=404, detail="Ruling not found")
+    return ruling
+
+@app.get("/search", response_model=List[Ruling])
+def search_rulings(query: str, db: Session = Depends(get_db)):
+    # A simple search implementation
+    search_query = f"%{query}%"
+    rulings = db.query(RulingDB).filter(
+        (RulingDB.question.like(search_query)) |
+        (RulingDB.answer.like(search_query)) |
+        (RulingDB.text.like(search_query))
```

Ruff (0.11.9)

Missing return type annotation for public function `search_rulings`

(ANN201)